tests/testthat/test-Logger.R

```diff
@@ -120,3 +120,29 @@ test_that("Logger works", { for (conn in conns) { # nolint: brace_linter
     purrr::keep(~ endsWith(., ".log")) |>
     purrr::walk(~ unlink(file.path(log_path, .)))
 }})
+
+test_that("Logger stops if file exists", { for (conn in conns){
```

Comment:
This line will likely give a linting issue.

I suggest to write instead:
test_that("Logger stops if file exists", { for (conn in conns) { # nolint: brace_linter